docs/puml/SequenceDiagrams/EditRecipes.puml

```diff
@@ -13,12 +13,11 @@ skinparam {
 box Editing Recipes COLOR_PALE_PINK
 participant "YMFC" as YMFC <<class>>
 participant "Parser" as Parser <<class>>
-participant ":EditCommand" as EditCommand
+participant "command:EditCommand" as EditCommand
 participant ":Storage" as Storage
 participant ":RecipeList" as RecipeList
 participant ":Storage" as Storage
 participant ":UI" as UI
```

Comment:
Maybe can initialise `:Storage` as `storage:Storage` to show how the storage object is referenced by its name, and similarly for recipeList and ui. Also, storage seems to be initialised twice here.